src/components/Buildbaron/BuildBaronTab.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { useQuery } from "@apollo/client";
+import { Tab } from "@leafygreen-ui/tabs";
+import BuildBaron from "components/Buildbaron/BuildBaron";
+import { BuildBaronQuery, BuildBaronQueryVariables } from "gql/generated/types";
+import { GET_BUILD_BARON } from "gql/queries";
+
+interface Props {
+  taskId: string;
+  execution: number;
+  setshowbuildbarontab: React.Dispatch<React.SetStateAction<Boolean>>;
```

Comment:
camel case please.

Reply:
I also dont think this needs to be passed in. The component can handle rendering itself.

Reply:
I need it to rerender when it changes.

Reply:
that setState will actually cause a render in the parent component by accessing a data from a request that was dispatched from the child. I think this variable and the useState hook that it originated from can be removed because leafygreen can now handle tabs being conditionally mounted/unmounted